rhizome/postgres/bin/install_postgres

```diff
@@ -3,15 +3,5 @@
 
 require_relative "../../common/lib/util"
 
-r "sh -c 'echo \"deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main\" > /etc/apt/sources.list.d/pgdg.list'"
-r "wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -"
-r "apt-get update"
-r "apt-get -y install postgresql-common"
-
-r "echo \"initdb_options = '--data-checksums'\" | sudo tee -a /etc/postgresql-common/createcluster.conf"
-r "echo \"create_main_cluster = 'off'\" | sudo tee -a /etc/postgresql-common/createcluster.conf"
-r "echo \"data_directory = '/dat/16/data'\" | sudo tee -a /etc/postgresql-common/createcluster.conf"
-
 r "apt-get -y install postgresql-16"
```

Comment:
Why don't we install postgresql-16 to the base image?

Reply:
I initially thought it would require /dat to be mounted to work properly, but it doesn't actually need that because we already set `create_main_cluster = 'off'`, so I moved it to the image.